.changelog/32464.txt

```diff
@@ -0,0 +1,11 @@
+```release-note:bug
+resource/aws_quicksight_*: Fix exception thrown when specifying `definition.sheets.visuals.bar_chart_visual.chart_configuration.category_axis.scrollbar_options.visible_range`
```

Comment:
If you'd like to break these out into explicit resources, the three which share this underlying schema are `aws_quicksight_template`, `aws_quicksight_dashboard`, and `aws_quicksight_analysis`.

Reply:
Perfect, thanks! I've pushed that change.